### ISSUE TITLE
Could GIT_TOGETHER_GLOBAL be an environment variable?

### ISSUE THREAD

**Issue by cwlbraa** [kejadlen/git-together#41]
see https://github.com/pivotal/git-author/issues/22 for a full reproduction of the real issue in our development environment. 

In multi-repo development environments (all of CF) we'd like to be able to set authors for all repos globally, every time. We also use git aliases heavily. git-author fits most of our needs, but leaves a small gap where using `git with ab cd` can set local repo state. Once that state is set, it frequently sticks around and sneaks into our commit messages. We'd like to effectively disable running `git with` without the "global" flag, and it seems like an environment variable would make that really easy.

**Comment by kejadlen** [comment]
Makes sense - I'd been wanting to add a configuration for this in git's config, rather than an environment variable, though. You could set `git-together.global` to `true` or something in the global `~/.gitconfig` to permanently turn it on and make it impossible to set a local git pair accidentally.
